stack.yaml

```diff
@@ -16,25 +16,28 @@ allow-newer: true
 
```

Comment:
BTW, this `allow-newer: true` changes all dependency pins into lower bounds, which *is not* the same thing.

Reply:
:+1: 
this is not needed anymore; removed.